inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/openapi/DataProxyController.java

```diff
@@ -56,7 +56,11 @@ public class DataProxyController {
     @ApiOperation(value = "Get data proxy IP list by InlongGroupId")
     public Response<DataProxyNodeResponse> getIpList(@PathVariable String inlongGroupId,
             @RequestParam(required = false) String protocolType) {
-        return Response.success(clusterService.getDataProxyNodes(inlongGroupId, protocolType));
+        try {
```

Comment:
There is no need to catch exceptions here, BusinessException will be handled in the ControlExceptionHandler.

Reply:
Thank you，I have removed it